stdlib/Std/src/WebSockets.luna

```diff
@@ -113,15 +113,15 @@ class WebSocket:
     # `wss` part of the url, you must explicitly call `setSecure` to override
     # the defaults. Example usage:
     # ```
-    # conn = WebSocket.connection "ws://somehost.com/sockets" . setPort 12345 . create
+    # conn = WebSocket.connection "ws://somehost.com/sockets" "/path". setPort 12345 . create
```

Comment:
This documentation is misleading now – `/sockets` is not a part of host.

Reply:
Also, we need to fix spacing in that snippet – the spacing around `.` in Luna is important – you need either spaces on both sides or no spacing, for a method call to work. Otherwise you end up with a lambda or parse error.